src/governance.ts

```diff
@@ -136,15 +136,15 @@ export class GovernanceCanister {
    */
   public getProposalInfo = async ({
     proposalId,
-    certified = true
+    certified = true,
   }: {
-    proposalId: bigint,
-    certified?: boolean
+    proposalId: bigint;
+    certified?: boolean;
   }): Promise<ProposalInfo | null> => {
     const service = certified ? this.certifiedService : this.service;
     const rawResponse = await service.get_proposal_info(proposalId);
     return rawResponse.length
       ? this.responseConverters.toProposalInfo(rawResponse[0])
```

Comment:
seems we have got this pattern "if length ? something[0] : null" a couple of times too, maybe worth to have a util functions for it or a TODO as reminder

Reply:
maybe in the `ResponseConverters` itself? This is because the response is an option, therefore it makes sense to have it there.